mods/ra/weapons/smallcaliber.yaml

```diff
@@ -266,7 +266,7 @@ M60mg:
 	Range: 2c512
 	Report: gun5.aud
 	ValidTargets: Ground, Infantry
-	InvalidTargets: Vehicle, Water, Structure, Wall, Husk
+	InvalidTargets: Vehicle, Water, Structure, Wall, Husk, Mine
```

Comment:
What speaks against letting Tanya fire at mines?

Reply:
Tania supposed to only target infantry and buildings. It can't target mines for the same reason it can't target vehicles/aircraft